tehom/downloads.py

```diff
@@ -32,6 +34,9 @@
 
 from . import _persistence
 
+# from zipfile import ZipFile
```

Comment:
delete this and extra lines

Reply:
This is in for unzip-ais, which I think i have since piushed
 
If not, that'll happen shortly